src/Components/WebAssembly/BlazorManifest/src/WorkloadManifest.json.in

```diff
@@ -0,0 +1,33 @@
+{
+  "version": 1,
+  "workloads": {
+    "microsoft-net-sdk-blazorwebassembly-aot": {
+      "description": "Blazor WebAssembly AOT workload",
```

Comment:
Are you sure you want to have a different workload pack for AOT and non-AOT configuration?

Reply:
I haven't done any work to make BlazorWebAssembly SDK an optional pack. It's still pat of the .NET installer. The way this works is that if a Blazor project enables AOT compilation, Blazor SDK wires up the AOT targets from these packs: https://github.com/dotnet/sdk/blob/master/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets#L559-L581. 

The workflow is a developer acquires the AOT workload (and emscripten), enables AOT in their Blazor Wasm project file (`RunAOTCompilation=true`) and publishes.

Does that align with how you think about this?

Reply:
> Does that align with how you think about this?

It does not. I see this as Blazor WebAssembly workload, not AOT specific configuration. The end-user scenario for me is that a developer installs .NET Sdk and then goes and installs blazor-wasm workload. The workload manifest for that will include all necessary dependencies to develop and possibly also publish blazor-wasm whether it's with AOT or without it. In other words most of the tooling will be required for non-aot or hot reload setup as well.

The scenario where you install the workload and disconnect from the internet is I believe also supported.